src/afl-fuzz-run.c

```diff
@@ -528,11 +535,21 @@ void sync_fuzzers(afl_state_t *afl) {
     closedir(qd);
     ck_free(qd_path);
     ck_free(qd_synced_path);
+    if (n > 0)
+      for (m = 0; m < n; m++)
+        free(namelist[m]);
+    free(namelist);
 
   }
 
   closedir(sd);
 
+  clock_gettime(CLOCK_REALTIME, &spec);
+  profile_end = (spec.tv_sec * 1000000000) + spec.tv_nsec;
+  sprintf(path, "echo %016llu %u >> /tmp/profile.out",
+          profile_end - profile_start, afl->queued_imported);
+  if (system(path) != 0) WARNF("system");
```

Comment:
Why not open, write, close? This seems like it would break eventually on random systems.

Reply:
this is just for profiling and testing how long it takes. nothing for production code.